package.json

```diff
@@ -96,7 +96,7 @@
     "tailwind-merge": "^2.5.4",
     "tailwindcss-animate": "^1.0.7",
     "use-keyboard-shortcut": "^1.1.6",
-    "xlsx": "^0.18.5"
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.3/xlsx-0.20.3.tgz"
```

Comment:
@Sancharisingh this change seems to be unrelated to the linked PR. If you meant to solve #9183, please raise a separate PR solving that alone.

Reply:
@rithviknishad, I’ve removed the changes related to SheetJS. Please review now and let me know if further adjustments are needed.